lib/tool_mount_point/wpcc.rb

```diff
@@ -0,0 +1,6 @@
+module ToolMountPoint
+  class Wpcc < Base
+    EN_ID = 'wpcc'
```

Comment:
This is not the feature 8191, it is the **[8460](https://moneyadviceservice.tpondemand.com/entity/8460)**
This is the user story to *deploy wpcc to a secret urll* and **not replace the current tool* yet.